### PR TITLE
purge docs and nox

### LINKED ISSUE #1552: Sunset `qhub.dev`
### Context

We can plan to remove the `docs` folder from this repo, and redirect `qhub.dev` to `nebari.dev`, after all the i[mportant docs pages are moved to `nebari.dev`](https://github.com/nebari-dev/nebari-docs/issues?q=is%3Aissue+is%3Aopen+label%3Aold-doc-migration+).

Timeline: By Q1 2023

### Value and/or benefit

One source of truth for documentation. :)

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by pmeier**
## Reference Issues or PRs

Closes #1552 modulo the redirection from [qhub.dev](https://qhub.dev) to [nebari.dev](https://nebari.dev). @costrouc Can you handle that?

## What does this implement/fix?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds a feature)
- [ ] Breaking change (fix or feature that would cause existing features not to work as expected)
- [ ] Documentation Update
- [ ] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] Other (please describe):

## Testing

- [ ] Did you test the pull request locally?
- [ ] Did you add new tests?

### CODE REVIEW

**Comment by pmeier** [comment]
I messed up something with the markdown formatting. I'm trying to roll that back.

File: README.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ on [HPC](#nebari-hpc) or [Kubernetes](#nebari) with minimal DevOps overhead.
 
 **This repository details the [Nebari](https://nebari.dev/ "Official Nebari docs") (Kubernetes) version.**
 
-Not sure what to choose? Check out our [Setup Initialization](docs/source/installation/setup.md) page.
```
Comment (pmeier) [inline README.md:50]: Old:

https://github.com/nebari-dev/nebari/blob/8bd56f90b84c7686f45459ae8f7cf690721bd60a/docs/source/installation/setup.md?plain=1#L1-L7

The closest equivalent would be https://www.nebari.dev/docs/get-started/deploy

Screenshot from 2023-05-10 23-04-41

LMK if you prefer this over linking to the general "Get started" page.

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -78,17 +78,3 @@ repos:
       - id: terraform_fmt
         args:
           - --args=-write=true
-
-  # markdown
-  - repo: https://github.com/executablebooks/mdformat
-    rev: 0.7.16
-    hooks:
-      - id: mdformat
-        files: ^docs/
```
Comment (pmeier) [inline .pre-commit-config.yaml:87]: I'm a little confused about this one. Don't we want to auto-format all markdown files? As is, we only formatted the ones in the `docs/` directory and thus this PR removes this hook.

File: README.md (hunk 1 above)
Comment (pavithraes) [reply README.md:50]: Yes, I think we can link to "Choosing a deployment platform" here.

We also have a link to "Get Started" later in the page :)

File: README.md [hunk 3]
```diff
@@ -47,7 +47,7 @@ on [HPC](#nebari-hpc) or [Kubernetes](#nebari) with minimal DevOps overhead.
...
+Not sure what to choose? Check out our [Get started](https://www.nebari.dev/docs/get-started) page.
```
Comment (pavithraes) [inline README.md:50]:
```suggestion
Not sure what to choose? Check out our documentation on [choosing a deployment platform](https://www.nebari.dev/docs/get-started/deploy)
```

File: .pre-commit-config.yaml (hunk 2 above)
Comment (pavithraes) [reply .pre-commit-config.yaml:87]: Interesting. 👀 

Other than `docs/`, only `README.md` and `RELEASE.md` files need to be formatted in this repo. All the other markdown files are being synced from `nebari-dev/.github` and get formatted in the `.gtihub` repo.

I'm comfortable removing this bit, or if you'd like, we can explicitly mention `README.md` and `RELEASE.md` here.

**Review by pavithraes — APPROVED** [review]
@pmeier Thank you for working on this!

I've replied to your comments. Since we only have optional+minor changes, I'll approve this PR and you can feel free to self merge when ready. :)

**Comment by pavithraes** [comment]
> the redirection from [qhub.dev](https://qhub.dev/) to [nebari.dev](https://nebari.dev/). `@costrouc` Can you handle that?

I'll open a new issue to track this. :)

File: .pre-commit-config.yaml (hunk 2 above)
Comment (pmeier) [reply .pre-commit-config.yaml:87]: > only `README.md` and `RELEASE.md` files need to be formatted in this repo.

Oh, ok, I didn't know that.

> I'm comfortable removing this bit, or if you'd like, we can explicitly mention README.md and RELEASE.md here.

No strong opinion. If there were no issues in the past, I'm happy to drop the formatter for now. Just FYI: both files are not formatted according to this hook. This is what happened in https://github.com/nebari-dev/nebari/pull/1801/commits/52060831eb9b599cf8f66515666fdc36c76e141b#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5

File: README.md (hunk 1 above)
Comment (pmeier) [reply README.md:50]: Done in https://github.com/nebari-dev/nebari/pull/1801#discussion_r1206493393.

**Pull request merged**